src/effects/EffectUI.cpp

```diff
@@ -501,6 +501,9 @@ void EffectUIHost::OnPaint(wxPaintEvent & WXUNUSED(evt))
 
 void EffectUIHost::OnClose(wxCloseEvent & WXUNUSED(evt))
 {
+   if (mPlaying)
```

Comment:
Conditional check is not needed as StopPlayback returns early if mPlaying is not set.